charts/argocd-apps/values-integration.yaml

```diff
@@ -497,6 +497,44 @@ govukApplications:
             key: bearer_token
       - name: PUBLISH_TOPICS
         value: "1"
+- name: govuk-replatform-test-app
+  repoName: govuk-replatform-test-app
```

Comment:
nit: `repoName` is redundant here since it defaults to the value of `name`, so for clarity we would omit it.

Reply:
ok, will remove it